setup.py

```diff
@@ -0,0 +1,107 @@
+"""A setup script for APEL.
+
+This script installs the APEL library, client, parsers and server. This
+should be similar to installing the RPMs for apel-lib, apel-client,
+apel-parsers, and apel-server, although there may be some differences.
+
+Usage: 'python setup.py install'
+
+Requires setuptools.
+"""
+
+from os import remove, path, makedirs
+from shutil import copyfile
+import sys
+
+from setuptools import setup, find_packages
+
+from apel import __version__
+
+
+def main():
+    """Called when run as script, e.g. 'python setup.py install'."""
+    if 'install' in sys.argv:
+        # Create temporary files with deployment names
+        copyfile('bin/client.py', 'bin/apelclient')
+        copyfile('bin/parser.py', 'bin/apelparser')
+        copyfile('bin/dbloader.py', 'bin/apeldbloader')
+        copyfile('bin/dbunloader.py', 'bin/apeldbunloader')
+        copyfile('bin/summariser.py', 'bin/apelsummariser')
+        copyfile('bin/retrieve_dns.py', 'bin/apelauth')
+
+        if not path.exists('/var/log/apel'):
+            makedirs('/var/log/apel')
+
+        if not path.exists('/var/run/apel'):
+            makedirs('/var/run/apel')
+
+    # conf_files will later be copied to conf_dir
+    conf_dir = '/etc/apel/'
+    conf_files = ['conf/client.cfg',
+                  'conf/summariser.cfg',
+                  'conf/unloader.cfg',
+                  'conf/loader.cfg',
+                  'conf/db.cfg',
+                  'conf/parser.cfg',
+                  'conf/auth.cfg']
+
+    # schema_files, update_scripts, accounting_files,
+    # and message_files will later be copied to data_dir
+    data_dir = '/usr/share/apel'
+    schema_files = ['schemas/client.sql',
+                    'schemas/server.sql',
+                    'schemas/server-extra.sql',
+                    'schemas/cloud.sql',
+                    'schemas/storage.sql']
+
+    update_scripts = ['scripts/update_schema.sql']
+
+    accounting_files = ['scripts/slurm_acc.sh', 'scripts/htcondor_acc.sh']
+
+    message_files = ['scripts/msg_status.py']
+
+    # log_rotate_files will later be copied to log_rotate_dir
+    log_rotate_dir = '/etc/logrotate.d'
+    log_rotate_files = ['scripts/apel-client']
+
+    # For 'python setup.py install' to 
```

Comment:
Not to that exactly as I only have it locally, but it should be the same as for SSM:

https://github.com/apel/ssm/blob/dev/setup.py#L61